be/src/common/config.h

```diff
@@ -498,13 +498,19 @@ CONF_Bool(enable_metric_calculator, "true");
 // Max consumer num in one data consumer group, for routine load.
 CONF_mInt32(max_consumer_num_per_group, "3");
 
+// Max pulsar consumer num in one data consumer group, for routine load.
+CONF_mInt32(max_pulsar_consumer_num_per_group, "10");
```

Comment:
Why not reuse the `max_consumer_num_per_group` with kafka ?

Reply:
https://pulsar.apache.org/docs/concepts-messaging/#subscription-types
This PR needs the support of cumulative acknowledgment. For partitioned topics, one consumer should only attach one partition. So the number of consumers should be equal to the number of partitions.
The default value of `max_consumer_num_per_group` is 3, which is OK for kafka with more than 3 partitions. But for pulsar with more than 3 partitions, the consumer would get failure. 
Since this value is a config item needs to restart the BE to take into effect, `max_pulsar_consumer_num_per_group` with a larger default value is added.